maintainers/flake-module.nix

```diff
@@ -143,6 +143,7 @@
             ''^src/libstore/common-protocol-impl\.hh$''
             ''^src/libstore/common-protocol\.cc$''
             ''^src/libstore/common-protocol\.hh$''
+            ''^src/libstore/common-ssh-store-config\.hh$''
```

Comment:
why not apply formatting?

Reply:
We don't yet have a plan for how we want to format existing files --- e.g. creating conflicts with PRs, cherry-picks between us and Lix, etc. makes for some real downsides.

Reply:
there is already a clang-format; I find this take a bit odd.
Format and move on my take.